cmd/server/app/options/options.go

```diff
@@ -86,7 +87,8 @@ func (o *ProxyRunOptions) Flags() *pflag.FlagSet {
 	flags.UintVar(&o.AgentPort, "agent-port", o.AgentPort, "Port we listen for agent connections on.")
 	flags.UintVar(&o.AdminPort, "admin-port", o.AdminPort, "Port we listen for admin connections on.")
 	flags.UintVar(&o.HealthPort, "health-port", o.HealthPort, "Port we listen for health connections on.")
-	flags.DurationVar(&o.KeepaliveTime, "keepalive-time", o.KeepaliveTime, "Time for gRPC server keepalive.")
+	flags.DurationVar(&o.KeepaliveTime, "keepalive-time", o.KeepaliveTime, "Time for gRPC agent server keepalive.")
+	flags.DurationVar(&o.MasterKeepaliveTime, "master-keepalive-time", o.MasterKeepaliveTime, "Time for gRPC master server keepalive.")
```

Comment:
Sorry should have caught this earlier. "master" is a term against our policies (https://github.com/kubernetes/community/blob/master/wg-naming/language-evaluation-framework.md#is-the-term-overtly-racist). Can we rename this to either frontend or KAS? (On a separate note we need to fix our code base for this)